include/benchmark/benchmark.h

```diff
@@ -459,6 +460,57 @@ inline BENCHMARK_ALWAYS_INLINE void DoNotOptimize(Tp& value) {
   asm volatile("" : "+m,r"(value) : : "memory");
 #endif
 }
+#else
+#if defined(BENCHMARK_HAS_CXX11) && (__GNUC__ >= 5)
```

Comment:
i think #elif simplifies this slightly and removes one of the nested #ifs

Reply:
Good point. I've added a fix.